src/python/module/z5py/dataset.py

```diff
@@ -195,7 +195,7 @@ def _create_dataset_zarr(path, dtype, shape, chunks,
                   'compressor': Dataset._to_zarr_compression_options(compression,
                                                                      compression_options)}
         with open(os.path.join(path, '.zarray'), 'w') as f:
-            json.dump(params, f)
+            json.dump(params, f, cls = get_json_encoder())
```

Comment:
Could you PEP8ify this pls:
Function keyword arguments should not have whitespaces:
Hence this should be 
```py
json.dump(params, f, cls=get_json_encoder())
```